### PR TITLE
Fix disabling autocomplete and fix completing special channels

### PR DESCRIPTION
**Pull request by xPaw**

### CODE REVIEW

File: client/js/autocompletion.js [hunk 1]
```diff
@@ -318,7 +318,7 @@ function completeChans(word) {
 		.each(function() {
 			const self = $(this);
 
-			if (!self.hasClass("lobby")) {
+			if (self.hasClass("channel")) {
```
Comment (astorije) [inline client/js/autocompletion.js]: This disables from queries as well, right? Is that intentional?

Comment (astorije) [reply client/js/autocompletion.js]: So this should be `if (self.hasClass("channel") || self.hasClass("query")) {` I think.

Comment (xPaw) [reply client/js/autocompletion.js]: But what reason do you have to complete queries after typing `#` somewhere?

Comment (astorije) [reply client/js/autocompletion.js]: Oh wait, I think I'm reading this incorrectly.

Comment (astorije) [reply client/js/autocompletion.js]: Yes for some reason I read this as the location it's autocompleted from (which window) and not what is being populated, my bad.

**Pull request merged**